### PR TITLE
feat(ColorRangeMap): Translate from MUI to bootstrap

### PR DESCRIPTION
**Pull request by labdala**
------------------------ General approach --------------------------------
- Use of NavDropdown.Item, which supports icons/images in contrast with Form.Control option;
- Introduction of Row, Col, and Container to show the colorIcons in multiple columns;
- NavBar main does not support icons (as far as we know). The fix to show the current selected item in the main window was using an overlay image with `pointer events: none` so the click pass through it;
- Use of z-index to make the dropdown bar be on the top of other icons.

---------------------------- Design -------------------------------------

-  Use OverlayTrigger instead a title for the `Min` and `Max` labels because bootstrap does not seem to support titles hanging on top of the box (like MUI does).
Another possibility is to use Card and Card.Header (see label on the left for Min) vs OverlayTrigger (see label on the right for Max): 
Screenshot from 2022-06-17 12-16-30

------------------------------------ Fix --------------------------

**Fix bug:** Avoid range size less than 1
Avoid triggering the warning `attempt to set zero width color range` from itkVtkViewer, which would cause the data to be inacessible even if the user set the range to be bigger.

### CODE REVIEW

File: src/Images/ColorMapIconSelector.jsx [hunk 1]
```diff
@@ -45,45 +71,43 @@ function ColorMapIconSelector(props) {
...
-            </Tooltip>
-          </MenuItem>
-        ))}
-      </Select>
-    </FormControl>
+        <NavDropdown
+          title=""
+          id="basic-nav-dropdown"
+          className="form-control categoricalDropDown"
+          value={currentColorMap()}
```
Comment (PaulHax) [inline src/Images/ColorMapIconSelector.jsx:86]: Does `value` do anything if we just have an overlay?

File: src/style.css [hunk 2]
```diff
@@ -87,6 +87,7 @@ code.uiContainer {
 
 .dropdown-menu {
   min-width: 300px !important;
+  z-index: 1100 !important;
```
Comment (PaulHax) [inline src/style.css:90]: Does it work without !important?  Can be nice to not use !important if some other rule needs to override this for some reason in the future.

Comment (labdala) [reply src/style.css:90]: Good point. It does work without important for min-width, but not for z-index. So I adjusted accordingly 😸

File: src/Images/ColorRangeInput.jsx [hunk 3]
```diff
@@ -52,16 +93,16 @@ function ColorRangeInput(props) {
...
   const rangeChanged = (minVal, maxVal) => {
-    const bounds = actorContext.colorRangeBounds.get(
-      actorContext.selectedComponent
-    )
+    const bounds = boundsSelected
     const rangeMin = minVal < bounds[0] ? bounds[0] : minVal
-    const rangeMax = maxVal > bounds[1] ? bounds[1] : maxVal
+    // avoid range size < 1
+    const rangeMaxAux = maxVal > bounds[1] ? bounds[1] : maxVal
+    const rangeMax = rangeMaxAux === 0 ? bounds[0] + 1 : rangeMaxAux
```
[the remark this answers is not in the document]
Comment (labdala) [reply src/Images/ColorRangeInput.jsx]: I actually will have to adapt the code for the astronaut case, I just realize. But it should work for the engine.nrrd as it is.

File: src/Images/ColorRangeInput.jsx [hunk 4]
```diff
@@ -52,19 +106,48 @@ function ColorRangeInput(props) {
...
+            range: [rangeMin, rangeMax]
+          }
+        })
       }
-    })
+    } else if (isNaN(minVal)) {
+      setminIntent('')
+    } else if (isNaN(maxVal)) {
+      setmaxIntent('')
+    }
```
Comment (PaulHax) [inline src/Images/ColorRangeInput.jsx:148]: I could be missing something, but should we have a big `else` with 2 internal `if`s?
```
} else {
  if (isNaN(minVal)) {
      setminIntent('')
    } 

  if (isNaN(maxVal)) {
      setmaxIntent('')
    }
}
```

**Review by PaulHax — APPROVED** [review]
These color range selectors are much nicer than before!

Also, this code is very clean!

**Pull request merged**